src/demo-app/tsconfig.json

```diff
@@ -22,8 +22,11 @@
       ]
     }
   },
-  "exclude": [
-    "*-aot.ts"
+  "files": [
+    "./demo-app-types.d.ts",
+    "./demo-app-module.ts",
+    "./system-config.ts",
+    "./main.ts"
```

Comment:
Just looked at it. The issue you described has been solved in Webstorm. 

I just noticed that if you create a file (that is never referenced) Webstorm will complain for example about `experimentalDecorator` support. I don't consider this is as a big issue. What are your thoughts?